src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderBackCompat.cs

```diff
@@ -13,6 +13,12 @@ public class RSACryptoServiceProviderBackCompat
     {
         private static readonly byte[] s_dataToSign = { 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0 };
 
+        private static readonly RSAParameters s_publicOnly = new RSAParameters
```

Comment:
`s_rsa2048PublicOnly`, please.  Or, just inline it to the zero-to-two methods that use it.